internal/genesis/foundational.go

```diff
@@ -2,113 +2,113 @@ package genesis
 
 // GenesisFNAccounts are the ECSDA accounts for the foundational nodes.
 var GenesisFNAccounts = [...]DeployAccount{
-	{Address: "0x35D29200aFC9A4cDC05166096059a042078CB53e", Public: "0x35D29200aFC9A4cDC05166096059a042078CB53e", BLSKey: "f82fcd7b5dc07f3dd0259701300bb6746a789f8e0e7d72ac696108e075ff89d"},
-	{Address: "0xe4a69826534aD3f6ec6E432474B0380E7F9a9C3d", Public: "0xe4a69826534aD3f6ec6E432474B0380E7F9a9C3d", BLSKey: "8128924a101798a1c07679c21ad30f1c30bddc672f31bc3077e7cb0db24b86a"},
+	{Address: "0x35D29200aFC9A4cDC05166096059a042078CB53e", BlsPriKey: "af96cd1118284a8a5916359b7fa566346f2017eadc0d3efcaf67cffe374bf724"},
+	{Address: "0xe4a69826534aD3f6ec6E432474B0380E7F9a9C3d", BlsPriKey: "77da1ed33a8b00b2a227902f53045ed15aa2af735cb17ddbe5d42bcb29cda363"},
 
 	// 0 - 9
-	{Address: "0x04c3636dF766ad2d3E74424c016842f5704FAE3A", Public: "0x04c3636dF766ad2d3E74424c016842f5704FAE3A", BLSKey: "5837f197c5e01316d47e9e4d0207cb58f9a5addafe59122e81b3c21ea3f96fee"},
-	{Address: "0x053515CC2CAae77F7e2F0A9C48A27c8f6D76E99d", Public: "0x053515CC2CAae77F7e2F0A9C48A27c8f6D76E99d", BLSKey: "6830748bd831edeb48c1c292923f279f2a81c5c382c77b9aebcfe63620704ac"},
-	{Address: "0x0850243810E77fC6261965d2F163d36628E77E05", Public: "0x0850243810E77fC6261965d2F163d36628E77E05", BLSKey: "a30e9d8999787b5acfd72af1702f166d668022b62b5c44b56a882d4f60dfb81"},
-	{Address: "0x08aB87F3A8EB0b69a833575B6400670f3F330302", Public: "0x08aB87F3A8EB0b69a833575B6400670f3F330302", BLSKey: "3a9825d9fca01395737165b872297774283738270bd5795da518fcc12dc006d4"},
-	{Address: "0x0d51F2d1EB1716F30c6f72673a4A89a0A10cdf64", Public: "0x0d51F2d1EB1716F30c6f72673a4A89a0A10cdf64", BLSKey: "37e97aed614f5086f2fcee8c3152725440066c03f76e0d916a3e567056d31744"},
-	{Address: "0x144B2Fd168147311f749B0f9573664676C333e2A", Public: "0x144B2Fd168147311f749B0f9573664676C333e2A", BLSKey: "2bae3aeb4653657c1b30e4204e8c093bbab8e3ba93b421325dc0a3f60e53549c"},
-	{Address: "0x22117D26611161b1b1f4EBB06C441aeeA102261c", Public: "0x22117D26611161b1b1f4EBB06C441aeeA102261c", BLSKey: "588ec71b9999a4007663432e8e3fa50aae972ebe3bad89148f42e588cf101701"},
-	{Address: "0x24A8cD56bABef297F1C7234F830362466d01ff5d", Public: "0x24A8cD56bABef297F1C7234F830362466d01ff5d", BLSKey: "50a40637369e7d9d704f063c2c818a53e628d087b8d5a57a9661f4017c5ba0"},
-	{Address: "0x25347d09373B2644191f1DC4beDEFEBE26a5b2d1", Public: "0x25347d09373B2644191f1DC4beDEFEBE26a5b2d1", BLSKey: "72d41ddbfadf9b74f8ec4d6bf2c911b90325bf5bf6620ddd891c78fcf1b358d7"},
-	{Address: "0x25441821ecA41DEc79578aAB866d3627A2e9BB9f", Public: "0x25441821ecA41DEc79578aAB866d3627A2e9BB9f", BLSKey: "570dce63c362a8222fd564516538c84c81fdbc3d8dea0bd11129e0080c9d6486"},
+	{Address: "0x04c3636dF766ad2d3E74424c016842f5704FAE3A", BlsPriKey: "ff889b96e38934c08ea158ce32fb94ec605180a4f665ed378aea9b9ac1c39320"},
```

Comment:
we don't want to expose blsprivate key to public? - we should save here https://github.com/harmony-one/secret_keys -